android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -82,6 +79,10 @@ public class ReactExoplayerViewManager extends ViewGroupManager<ReactExoplayerVi
     private static final String PROP_DEBUG = "debug";
     private static final String PROP_CONTROLS_STYLES = "controlsStyles";
 
+    private static final String CMD_SET_PLAYER_PAUSE_STATE = "setPlayerPauseState";
+    private static final String CMD_SEEK = "seek";
+    private static final String CMD_SET_VOLUME = "setVolume";
+    private static final String CMD_SET_FULLSCREEN = "setFullscreen";
```

Comment:
We have method `setFullScreen` in JS so this one is not working
```suggestion
    private static final String CMD_SET_FULLSCREEN = "setFullScreen";
```